components/Card/CardHeader.tsx

```diff
@@ -8,6 +8,7 @@ export type CardHeaderProps = {
 	className?: string;
 	lowercase?: boolean;
 	noBorder?: boolean;
+	onClick?: () => void;
```

Comment:
Do we use this?

Reply:
Yes. We add an expand/collapse header to the deposit/withdraw modal.
https://github.com/Kwenta/kwenta/pull/1824/files#diff-b664ab95209131e3939e86b84aae29750a4edb86ac17568e4053c549ccda565cR148